app/controllers/kapnismology/smoke_tests_controller.rb

```diff
@@ -24,5 +27,19 @@ def status(evaluations)
         :service_unavailable
       end
     end
+
+    def results(evaluations)
+      items = evaluations.as_json
+      {
+        _links: {
+          self: request.original_url,
+          profile: PROFILE_URL
+        },
+        passed: evaluations.passed?,
+        count: items.select { |e| e.has_key?(:passed) } .size,
+        trace_id: request.headers[TRACE_ID],
```

Comment:
Maybe should be using Trace.id if available instead because it is possible that the trace was not sent to the test, then zipkin rack middleware would create its own traceid